app/src/main/java/com/example/features/dashboard/dagger/MainActivityModule.java

```diff
@@ -12,40 +12,36 @@
 import com.example.tools.analytics.AnalyticsHelper;
 import com.example.tools.images.ImageLoader;
 import com.example.util.mvp.base.Mapper;
-import com.example.util.other.ViewHolderFactory;
+import com.example.util.view.ViewHolderFactory;
 import dagger.Module;
 import dagger.Provides;
 
 @Module
-public final class MainActivityModule {
-
-    private MainActivityModule() {
-        throw new AssertionError();
-    }
+public class MainActivityModule {
 
     @Provides
-    public static MainActivityViewState providesMainActivityViewState() {
```

Comment:
I just removed them from that whole module, as now it needs to be instantiated by DaggerMock with a constructor call in MainPresenterTest. 
a) I didn't check if DaggerMock works with static providers
b) I removed them for the whole module even though we only mock the presenter object for the sake of simplicity